scalarxmat44_neon.c

```diff
@@ -0,0 +1,116 @@
+#include <arm_neon.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+
+#define N 4
+
+int main() {
+
+  int are_equal = 1;
+  float lamda = 5.0;
+  int count = 0;
+
+  // alocate alligned memory for both A,B matrices
+  //(16 byte allignment because of NEON intrinsics)
+  float *A_memory = NULL;
+  float *B_memory = NULL;
+  if (posix_memalign((void **)&A_memory, 16, sizeof(float[N][N])) != 0) {
```

Comment:
same comment as https://github.com/VectorCamp/simd-examples/pull/6#discussion_r1526480211
no need for dynamic allocation in the case of so small matrices.

Reply:
I added it because in bigger matrices it would be of great help.
Should I remove it ?

Reply:
yes please, keep it simple for now